src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -626,13 +626,21 @@ def set(self, bucket, bucket_id, uid=None, versioning_state=None,
                 bucket = '/{}'.format(bucket)
 
             # Link bucket to new user:
+            params = {
+                'bucket': bucket,
+                'bucket-id': bucket_id,
+            }
+
+            accounts = RgwAccounts().get_accounts()
```

Comment:
remind me again, did we managed to convert the `get_accounts` implementation to API instead of CLI scraping (just to make it faster)